common/network/neighbor.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __COMMON_NETWORK_NEIGHBOR_H__
+#define __COMMON_NETWORK_NEIGHBOR_H__
+
+#include "common/network/iota_packet.h"
+
+typedef enum connection_type_e { TCP, UDP } connection_type_t;
+
+typedef struct {
+  socket_addr_t endpoint;
+  int socket;
+  connection_type_t connection_type;
+  unsigned int number_of_all_transactions;
+  unsigned int number_of_new_transactions;
+  unsigned int number_of_invalid_transactions;
+  unsigned int number_of_sent_transactions;
+  unsigned int random_transaction_requests;
+  int flagged;
```

Comment:
what does `flagged` mean?

Reply:
Took the neighbours fields from IRI, guess it means it could be removed soon